guides/ansible_engine/1-adhoc/README.md

```diff
@@ -0,0 +1,100 @@
+# Exercise 1 - Running Ad-hoc commands
+
+For our first exercise, we are going to run some ad-hoc commands to help you get a feel for how Ansible works.  Ansible Ad-Hoc commands enable you to perform tasks on remote nodes without having to write a playbook.  They are very useful when you simply need to do one or two things quickly and often, to many remote nodes.
+
+## Step 1.1 - Define your Inventory
+
+Define your inventory.  Inventories are crucial to Ansible as they define remote machines on which you wish to run commands or your playbook(s). In this lab the inventory is provided by your instructor. The inventory is an ini formatted file listing your hosts, sorted in groups, additionally providing some variables. It looks like:
+
+```bash
+[all:vars]
+ansible_user=student1
+ansible_ssh_pass=PASSWORD
+ansible_port=22
+
+[web]
+node1 ansible_host=11.22.33.44 ansible_user=ec2-user
+node2 ansible_host=22.33.44.55 ansible_user=ec2-user
+node3 ansible_host=33.44.55.66 ansible_user=ec2-user
+
+[control]
+ansible ansible_host=44.55.66.77 ansible_ssh_user=ec2-user
+```
+
+## Step 1.2 - Ping a host
+
+Let's start with something really basic - pinging a host.  The `ping` module makes sure our web hosts are responsive.
+
+```bash
+ansible web -m ping
+```
+
+## Step 2:
+
+Now let's see how we can run a good ol' fashioned Linux command and format the output using the `command` module.
+
+```bash
+ansible web -m command -a "uptime" -o
+```
+
+## Step 3:
+
+Take a look at your web node's configuration.  The `setup` module displays ansible facts (and a lot of them) about an endpoint.
+
+```bash
+ansible web -m setup
+```
+
+## Step 4:
```

Comment:
@gdykeman Why are we doing this in the ad-hoc commands guide? In the hands-on workshop exercises we keep it short and leave this for the next assignment. I don't see the point of doing this here and breaking with the rest of the content.